src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map((tab) => {
+          const { id, title } = tab;
+
+          return (
+            <li
+              className={cn(tab === selectedTab && 'is-active')}
```

Comment:
You can also create a variable inside map function 
`const isTabSelected = tab === selectedTab`

To avoid code duplication